api/layer/layer.go

```diff
@@ -75,6 +77,11 @@ type (
 		System      *cache.Config
 	}
 
+	// NeoFSConfig stores params for requests to NeoFS.
+	NeoFSConfig struct {
+		CopiesNumber uint32
```

Comment:
I suggest keep and handle this value in `handlers`. Like it's done with `placement_policy` https://github.com/nspcc-dev/neofs-s3-gw/blob/8eb1afbcb8d5385d1f9e15099c20b6708afdf50f/api/handler/api.go#L27